checkerScripts.js

```diff
@@ -116,6 +118,16 @@ var kingSelected = null;
 var hasNotJump = false;
 
 
+if(localStorage){
```

Comment:
Please add the public object `window` to `localStorage`, of that way we know that is a public variable of javascript 

**i.e.:**

``` javascript
window.localStorage
```

The same for the other places you are using it.